### ISSUE TITLE
Layout Mixup while listview/scrollview is not closed(or scroll it little  and open it up again).

### ISSUE THREAD

**Issue by DhavalsShah** [zhaozhentao/InboxLayout#4]
After open up the InboxLayoutListview/scrollview, if pull it down slowly and again pull it up, At that time the InboxBackgroundScrollView and InboxLayoutListView overlap on each other. Screenshot is attached for understanding purpose.  And let me know if any solution is available for this issue

![screenshot_2015-11-12-16-59-06](https://cloud.githubusercontent.com/assets/13002046/11117322/ef841a48-895e-11e5-9a67-f338d29d13a9.png)

**Comment by DhavalsShah** [comment]
Yes,Got a Solution! I have just change the background color of dragableView inside IndexLayoutListView class. 
 dragableView.setBackground(new ColorDrawable(Color.parseColor("#ffffff")));
hope this help someone who faces similar issue.

**Comment by DhavalsShah** [comment]
If any other solution is there. then comment below.

**Comment by zhaozhentao** [comment]
thank you

**Comment by ryandailey100** [comment]
Hi where does the code that DhavalsShah provided go in order to fix this bug?

**Comment by DhavalsShah** [comment]
@ryandailey100 In library, there is one class named InboxLayoutListView. In that class there is override method named createDragableView(Context context, AttributeSet attrs). Just replace code with below one. Hope this helps you.
``` Java
@Override
protected ListView createDragableView(Context context, AttributeSet attrs) 
{
        dragableView = new ListView(context);
        dragableView.setBackgroundColor(Color.parseColor("#ffffff"));
        dragableView.setId(android.R.id.list); dragableView.setDividerHeight(0);
        return dragableView;
}
```

**Comment by ryandailey100** [comment]
Thanks for the help! 👍
